langevals/evaluators/langevals/langevals_langevals/select_best_compare.py

```
@@ -0,0 +1,311 @@
+        if len(candidates) < 2:
+            return EvaluationResultSkipped(
+                details=(
+                    f"N-way compare needs at least 2 candidates with non-empty "
+                    f"outputs (got {len(candidates)})"
+                )
+            )
+
+        if self.settings.randomize_order:
+            seed = entry.row_index if entry.row_index is not None else 0
+            ordered = list(candidates)
+            random.Random(seed).shuffle(ordered)
```

Ruff (0.15.20)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)